### ISSUE TITLE
Migration Guide from HackMD CE to CodiMD

### ISSUE THREAD

**Issue by HermannDppes** [codimd/server#346]
As the HackMD CE version is still maintained under the name CodiMD, it would be really good to have two things:
* A comparison of the different features
* A guide to migrating

As the former is most likely even harder to come by than the latter, I want to focus on that: Let's say I was recommended to deploy a CodiMD, searched for it, took a good look at the first project that came up and concluded that this is obviously the right one. A while later, I notice that not only are there two projects with the same name but I have installed the wrong one! What to do?

# tl;dr

I have a working installation of HackMD CE/CodiMD. I'd like to switch to CodiMD. Is there a better way than „nuke it from orbit, install the other one“?

# Limitations

I'm aware that both projects are under active development and a final guide that won't need daily updates seems unreasonable but if you could provide some general guidelines in that direction, I'd be grateful.

**Comment by literarymachine** [comment]
> I have a working installation of HackMD CE/CodiMD. I'd like to switch to CodiMD. Is there a better way than „nuke it from orbit, install the other one“?

As of now, this should read *I'd like to switch to HedgeDoc*, right? At least that is what I would like to do, and it would be fantastic to have a feature comparison and a migration guide.

**Comment by HermannDppes** [comment]
Yes. When was that namechange? I was a bit confused to get a notification from a repo I had never heard of. :confused:

Neither [history.md](https://github.com/hedgedoc/hedgedoc/blob/develop/docs/content/history.md) nor [CHANGELOG.md](https://github.com/hedgedoc/hedgedoc/blob/develop/CHANGELOG.md) (which just contains a 404ing link) are helpful in figuring this out …

**Comment by DerMolly** [comment]
Oh, we need to update those files.

To answer your question:
The name was decided on in July '20 and after the new logo in September '20, we released 1.7 with the new name in December '20. So not so long ago.

We also documented it on our new website https://hedgedoc.org/history/ among other things…

**Comment by Exagone313** [comment]
Also, latest CodiMD is 2.x but HedgeDoc 2.x is not out yet. It is unclear if there is a supported upgrade path.

**Comment by SISheogorath** [comment]
From the [last stories](https://community.hedgedoc.org/t/solved-upgrade-from-dockerlized-codimd/271/2) we've seen, one can use HedgeDoc 1.x as drop-in replacement for CodiMD 2.x.

Also there will be a migration path from HedgeDoc 1.x to 2.0. But we can and will not guarantee any migration from CodiMD 2.X to HedgeDoc 2.X since both projects are developed completely independent.
